src/core/Table/columns/actionColumn.tsx

```diff
@@ -0,0 +1,70 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { HeaderProps } from 'react-table';
+import { Checkbox } from '../../Checkbox';
+import SvgColumnManager from '@itwin/itwinui-icons-react/cjs/icons/ColumnManager';
+import { DropdownMenu } from '../../DropdownMenu';
+import { IconButton } from '../../Buttons/IconButton';
+import { MenuItem } from '../../Menu';
+export const ACTION_CELL_ID = 'iui-table-action';
+
+export const ActionColumn = <T extends Record<string, unknown>>(
+  hasColumnManager: boolean,
```

Comment:
Just randomly thought... Would we expect here more props in future and would it make sense to have an object instead..
But maybe not worth it as barely we will add that many props here :/

Reply:
I agree this should be an object, even if we don't add anything in the future. A boolean parameter makes the usage really hard to read

```ts
ActionColumn(true)
```

vs

```ts
ActionColumn({ hasColumnManager: true })
```

Reply:
changed to an object